src/function.rs

```diff
@@ -123,3 +124,15 @@ fn env_var_or_default(
     Ok(value) => Ok(value),
   }
 }
+
+fn just_executable(_context: &FunctionContext) -> Result<String, String> {
+  let exe_path =
+    std::env::current_exe().map_err(|e| format!("Error getting just executable: {}", e))?;
```

Comment:
How about `Error getting current executable: {}`. Function error messages are printed, it will refer to the function name, which will mention `just_executable`.